configs/odoo/initializer/product_variant/drugs.csv

```diff
@@ -121,6 +121,7 @@ init.79c12ca2-0755-4bf4-b509-8c72239536f9,Ringer Lactate,product.categ_products_
 init.0f4627ca-7bb4-414f-a07e-5d24c0d3a442,Salbutamol Co 4 mg,product.categ_products_essential_drugs_orders,product,init.6fbbb779-94ca-4d83-8e02-8a559627b9fb,init.6fbbb779-94ca-4d83-8e02-8a559627b9fb,Ordered quantities
 init.3099b446-010e-426a-ba02-bf62f1077f2e,Salbutamol Pompe,product.categ_products_drug_orders,product,init.1957f9ad-2d7c-4bc3-ac8e-527849c7bd98,init.1957f9ad-2d7c-4bc3-ac8e-527849c7bd98,Ordered quantities
 init.d39b535c-9a32-49c1-b339-e512d811c44d,Salbutamol Sirop 2 mg/5 ml,product.categ_products_drug_orders,product,init.e0f3730a-8fdb-436f-9ed7-1a724801ecc5,init.e0f3730a-8fdb-436f-9ed7-1a724801ecc5,Ordered quantities
+init.8f9e67c9-816b-4beb-9c23-b275fddfd277,Serum de rehydratation oral,product.categ_products_essential_drugs_orders,product,init.1fd95743-5a10-4c67-8626-a4fd1a4c620a,init.1fd95743-5a10-4c67-8626-a4fd1a4c620a,Ordered quantities
```

Comment:
@kazlaw when do we use `product.categ_products_drug_orders` vs `product.categ_products_essential_drugs_orders`?

Reply:
Followed this https://github.com/mekomsolutions/odoo-config-c2c/pull/15/files#:~:text=init.8f9e67c9%2D816b%2D4beb%2D9c23%2Db275fddfd277%2CSerum%20de%20rehydratation%20oral%2Cproduct.categ_products_essential_drugs_orders%2Cproduct%2Cinit.1fd95743%2D5a10%2D4c67%2D8626%2Da4fd1a4c620a%2Cinit.1fd95743%2D5a10%2D4c67%2D8626%2Da4fd1a4c620a%2COrdered%20quantities

Reply:
Not so certain of the answer to your question